material-ui/README.md

```diff
@@ -0,0 +1,19 @@
+# Example app with [Material UI](https://mui.com/)
+
+This example features how to use [Material UI](https://mui.com/) as the component library within a Remix app.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/examples/tree/main/material-ui)
+
+## Example
+
+A fork of the MUI remix example having two routes. [Extra steps are needed to support SSR for MUI](https://mui.com/material-ui/guides/server-rendering/).
```

Comment:
Could you expand this description a bit and reference relevant files like `client.entry.tsx`, etc ?
This should explain the implementation to the end user.

Reply:
Also, does this example support SSR or not? The link makes it seem like it's not the case.